tests/testthat/test-classify-diabetes.R

```diff
@@ -157,3 +157,38 @@ test_that("classifying works for data.table", {
   # expect_identical(colnames(actual), expected_columns)
   # expect_identical(actual_rows, expected_rows)
 })
+
+test_that("casing of input variables doesn't matter", {
+  # Testing this for data.frame input only here.
+  registers_as_df <- register_data |>
+    purrr::map(as.data.frame) |>
+    # Convert column names to upper case.
+    purrr::map(~ setNames(.x, toupper(names(.x))))
+
+  actual <- classify_diabetes(
+    kontakter = registers_as_df$kontakter,
+    diagnoser = registers_as_df$diagnoser,
+    lpr_diag = registers_as_df$lpr_diag,
+    lpr_adm = registers_as_df$lpr_adm,
+    sysi = registers_as_df$sysi,
+    sssy = registers_as_df$sssy,
+    lab_forsker = registers_as_df$lab_forsker,
+    bef = registers_as_df$bef,
+    lmdb = registers_as_df$lmdb
+  ) |>
+    dplyr::collect()
```

Comment:
```suggestion
  )
```

Collect only matters if it is through a database connection. But this is a dataframe so no need for collect.